open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -34,7 +34,6 @@
     "* Kernel Density Estimation Plots\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            assert_df["c2"] = pd.DataFrame(assert_df["c2"], dtype=int)
</pre><p>why create a df again? you can use the assert df directly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done